shared_workflow/platform_configs/platform_local.json

```diff
@@ -12,6 +12,7 @@
   "PLATFORM_ACCOUNTS": ["nesi00213", "comm00213"],
   "DEFAULT_MEMORY" : "16G",
   "SCHEDULER": "bash",
+  "VM_PERT_DEFAULT_NCORES":1,
```

Comment:
It's default set to 4 in the slurm script. Should we keep that?

Reply:
this is for local runs, which i doubt anyone would use?

Reply:
and all other values are set to 1 as well.

Reply:
i personally think its fine to keep it as one, as if anyone is using this config, that means
1. the workflow failed to identify the machine, thus safer to assume its serial
2. advanced user know what he is doing, thus should be able to adjust the config to required.